app/assets/javascripts/modules/cookie-settings.js

```diff
@@ -2,11 +2,11 @@ window.GOVUK = window.GOVUK || {}
 window.GOVUK.Modules = window.GOVUK.Modules || {};
 
 (function (Modules) {
-  function CookieSettings () {}
-
-  CookieSettings.prototype.start = function ($module) {
-    this.$module = $module[0]
+  function CookieSettings ($module) {
+    this.$module = $module
+  }
 
+  CookieSettings.prototype.init = function ($module) {
```

Comment:
Does `$module` need to be passed in here? I thought `this.$module` that's set in the constructor would be used in this function?

Reply:
Shouldn't be here, no.

Reply:
Oops! good catch